packages/utils/package.json

```diff
@@ -78,6 +78,10 @@
   },
   "devDependencies": {
     "@typescript-eslint/parser": "6.2.0",
+    "downlevel-dts": "*",
+    "jest": "29.6.1",
```

Comment:
Having to match all these `jest` versions is pretty cumbersome, but required to make the naked `jest` calls in npm scripts work with modern yarn.

The way they recommend handling this is different and involves sharing scripts between workspaces, or using `run -T` in front of the script contents.

Somewhat related - we could avoid the need for this entirely if we always let Nx run the tasks through its own executors and then just switch the script contents to be `nx test` like we did here: https://github.com/typescript-eslint/typescript-eslint/blob/ed8c6247c0454f84b126f5fbe16fe1c40f6605f3/packages/scope-manager/package.json#L43 - it would resolve `jest` for us, and we wouldn't need to bend over backwards to make yarn happy

Reply:
That sounds good to me! IIRC you'd mentioned this as a potential followup a while back. I'm definitely in favor.